composer.json

```diff
@@ -14,35 +14,35 @@
     "require-dev": {
         "spryker/app-payment": "^2.0.0",
         "spryker/app-webhook": "^1.0.0",
-        "codeception/codeception": "^5.0",
+        "codeception/codeception": "^5.0.0",
         "codeception/module-asserts": "^3.0.0",
         "codeception/module-cli": "^2.0.0",
         "codeception/module-filesystem": "^3.0.0",
         "codeception/module-phpbrowser": "^3.0.0",
         "codeception/module-rest": "^3.0.0",
         "codeception/module-webdriver": "^3.0.0",
         "codeception/phpunit-wrapper": "^9.0.0",
-        "galbar/jsonpath": "^3.0",
+        "galbar/jsonpath": "^3.0.0",
         "infection/codeception-adapter": "^0.4.3",
         "infection/infection": "^0.29.6",
         "phpstan/phpdoc-parser": "1.25.0",
         "phpstan/phpstan": "1.10.66",
         "rector/rector": "^0.19.0",
-        "spryker-sdk/async-api": "dev-master as 0.3.1",
+        "spryker-sdk/async-api": "^0.3.2",
         "spryker/app-kernel": "^1.0.0",
         "spryker/code-sniffer": "*",
         "spryker/container": "*",
         "spryker/development": "^3.34.0",
-        "spryker/event-dispatcher": "^1.6",
+        "spryker/event-dispatcher": "^1.6.0",
         "spryker/glue-backend-api-application": "*",
-        "spryker/glue-json-api-convention": "^1.3",
-        "spryker/http": "^1.11",
+        "spryker/glue-json-api-convention": "^1.3.0",
+        "spryker/http": "^1.11.0",
         "spryker/message-broker-aws": "*",
         "spryker/propel": "*",
-        "spryker/router": "^1.19",
+        "spryker/router": "^1.19.0",
```

Comment:
potential conflict with https://github.com/spryker/app-merchant/pull/9/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34

Reply:
Any action from my end you see here?

Reply:
synchronize packages list in require and required-dev